server/server/src/lib.rs

```diff
@@ -198,7 +199,7 @@ async fn run_server(
 
     let mut synchroniser = Synchroniser::new(sync_settings, service_provider_data.clone()).unwrap();
     // Do the initial pull before doing anything else
-    match synchroniser.initial_pull().await {
+    match synchroniser.sync().await {
```

Comment:
> sync would push and pull wouldn't it

Yep there is a check in synchroniser to only push when  initialise

>  this would delays the server startup time 

Server should not rely on sync when starting up, it is now and it's [causing the core problem](https://github.com/openmsupply/open-msupply/issues/122) that we were trying to solve with synchronisation. There is design of how we should do initialisation [here btw](https://github.com/openmsupply/open-msupply/issues/66)

> initial_pull would return immediately if the initial pull was already done.

In current implementation (even before this PR) I think we go back to bootstrap mode if central server is unreachable

Reply:
it was suppose to startup if the central server is not reachable (don't know if there was a bug in the latest version).